package.json

```diff
@@ -38,10 +38,10 @@
     "marked": "^0.3.2",
     "mocha": "^2.5.3",
     "mocha-loader": "^1.0.0",
-    "mocha-webpack": "^0.4.0",
+    "mocha-webpack": "next",
```

Comment:
No, just ran command from it's installation guide https://github.com/zinserjan/mocha-webpack#installation